packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -45,6 +44,7 @@ export interface TimeRangePickerProps {
   hideQuickRanges?: boolean;
   widthOverride?: number;
   isOnCanvas?: boolean;
+  onTimePickerClick?: () => void;
```

Comment:
I would change it to `onToolbarButtonClick`

Reply:
Would it make more sense to do something like `onToolbarTimePickerClick ` to differentiate from the other clicks on DashNav component? I was thinking doing  `onToolbarRefreshClick ` and  `onToolbarZoomClick ` as well

Reply:
yeap, makes sense!